### DISCUSSION TITLE
Is it possible to found somewhere old version of ImageMagick ?

### DISCUSSION THREAD

**Discussion by Zeus64** [ImageMagick/ImageMagick#5536] [General]
Hello, I m looking for ImageMagick-7.1.0-8-Q16-x86-dll binaries for windows (not the source code, the compiled package). Is their any way to retrieve it somewhere ?

**Comment by dlemstra** [comment]
We don't have the resources to keep long time backups so we cannot provide you with that version. Our advise would be to use the most recent release.

**Reply by Zeus64** [reply]
Thanks @dlemstra.

**Comment by fmw42** [comment]
Did you search Google?  I found https://www.wingetgui.com/apps?id=ImageMagick.ImageMagick&v=7.1.0.8 Does that help?

**Comment by Zeus64** [comment]
thanks [fmw42](https://github.com/fmw42), yes i already searched, but found nothing (the link you gave me return not found when I try to download) :(

**Reply by fmw42** [reply]
Sorry!
